Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "parse_datetime"
-description = " parsing human-readable relative time strings and converting them to a Duration"
+description = "parsing human-readable relative and absolute time strings and converting them to a DateTime"
```

```suggestion
description = "parsing human-readable time strings and converting them to a DateTime"
```
Since the relative/absolute distinction is no longer that important. I'd phrase it like this.